src/index.html

```diff
@@ -6,9 +6,36 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html" class="logo">
+        <img
+        src="/src/images/logo.png"
+        alt="Moyo"
+        class="logo__img">
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav__link"><a href="index.html" class="link is-active">Apple</a></li>
```

Comment:
Need to fix code style everywhere, also fix the classes everywhere
```suggestion
          <li class="nav__item">
            <a href="index.html" class="nav__link is-active">Apple</a>
          </li>
          
```